app/views/spree/shared/_item_info.html.erb

```diff
@@ -1,24 +1,30 @@
 <%
```

Comment:
I'm not sure about moving this partial into the `shared` folder. This folder should be removed in the future, right? (ref #30)

Reply:
Good catch!

Reply:
Yes, you're right. It's difficult to name a component that you know will be reused in multiple contexts. Should I place it under `components/orders` and reuse it from there?